### PR TITLE
Dependency Checker will parse CSS files

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Other: Introduced a parser for `*.css` files. Now it will check CSS files and if some package is missing, it will be printed by the dependency checker script. See ckeditor/ckeditor5#1662.

### CODE REVIEW

File: packages/ckeditor5-dev-tests/bin/check-dependencies.js [hunk 1]
```diff
@@ -155,3 +165,34 @@ function groupMissingPackages( missingPackages, currentPackage ) {
...
+ * Checks whether all packages that have been imported by the CSS file are defined in `package.json` as `dependencies`.
+ * Returned array contains list of missing packages.
+ *
+ * @param {String} fileContent Content of the checking file.
+ * @param {String} filePath An absolute path to the checking file.
+ * @param {Array.<String>>} dependencies Merged list of dependencies and devDependencies.
+ * @returns {Array.<String>|undefined}
+ */
+function parsePostCSS( fileContent, filePath, dependencies ) {
+	const matchedImports = fileContent.match( /^@import "(@ckeditor\/[^/]+)[^"]+";/mg );
```
Comment (Reinmar) [inline packages/ckeditor5-dev-tests/bin/check-dependencies.js]: Why only `@ckeditor/*` packages?

File: packages/ckeditor5-dev-tests/bin/check-dependencies.js [hunk 2]
```diff
@@ -155,3 +165,34 @@ function groupMissingPackages( missingPackages, currentPackage ) {
...
 	return { dependencies, devDependencies };
 }
+
+/**
+ * Checks whether all packages that have been imported by the CSS file are defined in `package.json` as `dependencies`.
+ * Returned array contains list of missing packages.
+ *
+ * @param {String} fileContent Content of the checking file.
+ * @param {String} filePath An absolute path to the checking file.
+ * @param {Array.<String>>} dependencies Merged list of dependencies and devDependencies.
```
Comment (ma2ciek) [inline packages/ckeditor5-dev-tests/bin/check-dependencies.js]:
```suggestion
 * @param {Array.<String>} dependencies Merged list of dependencies and devDependencies.
```

File: packages/ckeditor5-dev-tests/bin/check-dependencies.js (hunk 1 above)
Comment (pomek) [reply packages/ckeditor5-dev-tests/bin/check-dependencies.js]: Now it checks everything.

Comment (pomek) [reply packages/ckeditor5-dev-tests/bin/check-dependencies.js]: It means – relative file imports, scoped and non-scoped packages.

File: packages/ckeditor5-dev-tests/bin/check-dependencies.js [hunk 3]
```diff
@@ -135,7 +163,7 @@ function getInvalidItselfImports( repositoryPath ) {
  *
  * @param {Object} missingPackages The `missing` value from object returned by `depcheck`.
  * @param {String} currentPackage Name of current package.
- * @returns {Object.<String, Array.<String>>}
+ * @returns {Object.<String, Array.<String>}
```
Comment (ma2ciek) [inline packages/ckeditor5-dev-tests/bin/check-dependencies.js]:
```suggestion
 * @returns {Object.<String, Array.<String>>}
```

😄

**Pull request merged**